### PR TITLE
Makes visible rows selection persist

### LINKED ISSUE #2427: Make the selection of visible items per page persist
The default is 10 rows per page.  I can't see why anyone would ever want to click through pages with only 10 rows each, when scrolling is much easier.  I'd suggest increasing the default to 100 or more.  Also, it might be nice if the selected value was sticky so you don't have to change it every time you switch pages.  This applies to the Dashboard, as well as other pages that have rows.

### Task
- [x] Remember the user's selection across all the tables in the session
- [x] Remember the user's selection across different sessions.

### PR DESCRIPTION
**Pull request by kelsos**
Closes #2427
Also has premium changes

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`2427` The visible row selection will no persist after a re-login. Changing the visible rows will now affect all the tables.
```
Comment (LefterisJP) [inline docs/changelog.rst:5]:
```suggestion
* :feature:`2427` The visible row selection will now persist after a re-login. Changing the visible rows will now affect all the tables.
```

**Review by LefterisJP — COMMENTED** [review]
Does this need an `npm ci`?

I tried to change the ethereum transaction tables to 100 rows per page. Works. Then restarted, but it's still back to the default of 10. Am I misunderstanding something?

**Comment by LefterisJP** [comment]
> Does this need an `npm ci`?
> 
> I tried to change the ethereum transaction tables to 100 rows per page. Works. Then restarted, but it's still back to the default of 10. Am I misunderstanding something?

wait wait wait. Scratch that. Wrong branch. Let me test again.

**Review by LefterisJP — APPROVED** [review]
Okay seems to work. Approving with note for changelog. Not sure I like the stickiness of this for every table, but I guess it's the simplest approach to achieve persistence.

**Comment by kelsos** [comment]
> Okay seems to work. Approving with note for changelog. Not sure I like the stickiness of this for every table, but I guess it's the simplest approach to achieve persistence.

I was thinking that we can maybe make it per table in the future. But that would require some extra effort so that we can identify each unique table.

**Pull request merged**